ci/publishing-images.md

```diff
@@ -9,63 +9,63 @@ Many of our repositories, such as [Containers-Quickstarts](https://github.com/re
 In order to publish images to Quay.io, you'll need the following set up ahead of time:
 
 * A new repository and robot account created in Quay.io, set as a Secret on your repo. You can [open an issue with the CoP Tooling team](https://github.com/redhat-cop/org/issues/new?assignees=&labels=integrations&template=integrations.md&title=) to have this created.
-  * QUAY_USERNAME
-  * QUAY_PASSWORD
+  * REGISTRY_URI: quay.io
+  * REGISTRY_REPOSITORY: redhat-cop
```

Comment:
This is not entirely true as the repository includes both the `<organization/name>`

Reply:
suggest `REGISTRY_NAMESPACE`